pkgs/build-support/kernel/make-initrd-ng/src/main.rs

```diff
@@ -85,12 +83,25 @@ fn copy_file<P: AsRef<Path> + AsRef<OsStr>, S: AsRef<Path>>(
             }
         }
         if !found {
-            // glibc makes it tricky to make this an error because
+            // glibc makes this tricky because
             // none of the files have a useful rpath.
-            println!("Warning: Couldn't satisfy dependency {} for {:?}", line, OsStr::new(&source));
+            if line != "libc.so.6" && !line.starts_with("ld-linux-") {
```

Comment:
Isn't this to specific of a condition? I think any glibc lib depending on another will trigger the problem. Also, is that `6` number permanent or will it vary from libc to libc?

Reply:
It's the ABI version but I agree we can make this less specific

Reply:
Plus, supporting `musl` isn't exactly a priority by any means, but if we can avoid making `musl` support *harder*, that'd be nice (I *think* the current implementation would work with musl?)